src/main/java/net/snowflake/ingest/utils/Utils.java

```diff
@@ -354,9 +353,7 @@ public static void showMemory() {
 
     Runtime runtime = Runtime.getRuntime();
     logger.logInfo(
-        "Max direct memory={}, max runtime memory={}, total runtime memory={}, free runtime"
-            + " memory={}",
-        PlatformDependent.maxDirectMemory(),
+        "Max max runtime memory={}, total runtime memory={}, free runtime" + " memory={}",
```

Comment:
And any chance we could keep the direct memory?
```suggestion
        "Max runtime memory={}, total runtime memory={}, free runtime" + " memory={}",
```

Reply:
Unfortunately the `PlatformDependent` class comes from Netty, and this is the only direct usage of the Netty library. With the Netty framework including a number of libraries, some of which are specific to particular architectures, adding a direct dependency on Netty does not seem worth it logging. The previous approach built on the shaded copy from the Snowflake JDBC driver, but that is no longer an option.

Reply:
Pushed a correction to the log message as noted.